src/middlewared/middlewared/plugins/network.py

```diff
@@ -1149,17 +1149,36 @@ async def _common_validation(self, verrors, schema_name, data, itype, update=Non
                             'A lagg interface using the "Failover" protocol is not allowed to be marked critical for failover.'
                         )
 
-            if update.get('failover_vhid') != data['failover_vhid']:
```

Comment:
Isnt this the same? In a new interface, data['failover_vhid'] will be null and failover_vhid will be XX?

Reply:
A new interface will have data['failover_vhid'] as XX and update object will be NULL.

However, you're correct in the fact that the logic is the same. This commit is doing nothing but adding documentation (via comments) so I'll close.